src/elements/content-sidebar/SidebarNav.js

```diff
@@ -72,6 +84,7 @@ const SidebarNav = ({ hasSkills, hasMetadata, hasActivityFeed, hasDetails, onTog
                 <IconMetadataThick />
             </SidebarNavButton>
         )}
+        {hasAdditionalTabs && additionalTabs && <AdditionalTabs tabs={additionalTabs} />}
```

Comment:
Are we going to need an additional isLoading property to handle showing the loading state?

Reply:
my thought was that if the hasAdditionalTabs prop is set to true:
1. If we don't have any additional tabs data we are loading
2. If we have valid data load it
3. If we have an empty array then render the empty state

Reply:
Sounds good!